BuildaGitServerTests/GitHubServerTests.swift

```diff
@@ -14,11 +14,13 @@ import BuildaUtils
 class GitHubSourceTests: XCTestCase {
 
     var github: GitHubServer!
+    var enterpriseGithub: GitHubServer!
```

Comment:
It might be worth separating EnterpriseGithub tests into a separate file and only test the things not already tested in GitHub tests.